src/modules/BlobAnalysis/BlobAnalysis.js

```diff
@@ -0,0 +1,80 @@
+module.exports = function(pixels, options, priorStep){
+
+  var cv = require('../../../examples/opencv');
```

Comment:
So we'd change this line to `../../../node_modules/opencv.js/build/cv.js` I think?